alibi_detect/utils/pytorch/distance.py

```diff
@@ -30,6 +30,28 @@ def squared_pairwise_distance(x: torch.Tensor, y: torch.Tensor, a_min: float = 1
     return dist.clamp_min_(a_min)
 
 
+def squared_distance(x: torch.Tensor, y: torch.Tensor, a_min: float = 1e-30) -> torch.Tensor:
```

Comment:
Can we just apply a reduction to the `squared_pairwise_distance` instead of using an extra function?

Reply:
Now implemented as a single function.